hack/verify-conftest.sh

```diff
@@ -42,8 +42,8 @@ function main() {
         infra/gcp/clusters/projects/*/*/resources
     )
     local conftest_flags=(
-        # override the default of looking for $(pwd)/policy
-        --policy "${REPO_ROOT}/policies/"
+        # be explicit about looking for $(pwd)/policy
+        --policy "${REPO_ROOT}/policy"
```

Comment:
To match the new `area/policy` label, I relocated `policies/*` to `policy/` which should just work by default, but I felt like staying explicit was better